arbitrum-docs/node-running/how-tos/running-a-sequencer-coordinator-manager.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: 'How to run a Sequencer Coordinator Manager'
+description: Learn how to run a Sequencer Coordinator Manager terminal UI tool on your local machine.
+sidebar_label: Run a Sequencer Coordinator Manager
+sidebar_position: 10
+content-type: how-to
+---
+
+The Sequencer Coordinator Manager is a command-line tool that allows you to manage the priority list of sequencers, update their positions, add new sequencers to the list, and refresh the lists from the Redis server.
+The tool offers keyboard-only support for a seamless user experience during cases with no mouse support. Any changes you make are stored locally until you choose to save and push them to the Redis server. 
+
+- Here is an example of how to start the Sequencer Coordinator Manager and connect to a local redis server:
+  ```shell
+  git clone --branch @nitroVersionTag@ https://github.com/OffchainLabs/nitro.git
+  cd nitro
+  make target/bin/seq-coordinator-manager
+  ./target/bin/seq-coordinator-manager redis://127.0.0.1:6379
+  ```
+
+If mouse support is enabled, you can seamlessly navigate throughout the tool and perform any desired operations using mouse clicks. However, if mouse support is not enabled, mouse clicks are replaced by `enter`. You can navigate within a list using the up and down arrow keys or the `Tab` key. To change or switch the focus between lists, simply press `c` from the keyboard shortcuts.
```

Comment:
```suggestion
If mouse support is enabled, you can use your mouse to explore the tool. Otherwise, use your keyboard to explore the UI. The `enter` key selects options; `c` switches focus between lists.
```